openedxscorm/static/js/src/scormxblock.js

```diff
@@ -57,9 +57,15 @@ function ScormXBlock(runtime, element, settings) {
             if (settings.block_height > screen.height) {
                 $(e.target).css("height", screen.height);
             }
+            // Override height and cover entire display in fullscreen
```

Comment:
I think we should remove all this code and use `requestFullscreen` and `exitFullscreen` methods as described [here](https://www.w3schools.com/jsref/met_element_requestfullscreen.asp)